### ISSUE TITLE
Icon for emergency = access_point

### ISSUE THREAD

**Issue by A-d-r-i** [gravitystorm/openstreetmap-carto#4351]
Hi,
A little suggestion :
### Expected behavior
I intend to map the access points to French emergency services. It would be interesting to create an icon representing the sites in order to quickly detect them on a map.

Tag : [emergency=access_point](https://wiki.openstreetmap.org/wiki/Tag:emergency%3Daccess_point) with [~7000 objects](https://taginfo.openstreetmap.org/tags/emergency=access_point)

### Actual behavior
No particular icon, a simple white mark in the editor which becomes invisible on the final map ([example](https://www.openstreetmap.org/node/4938403421))

Thanks,
A-d-r-i

**Comment by jeisenbe** [comment]
The more common tag for this feature is highway=emergency_access_point - used 37k times:

https://wiki.openstreetmap.org/wiki/Tag:highway%3Demergency_access_point

But neither of these are information that you would expect to find on a general-purpose map. What would be the use of rendering this for general map users?

**Comment by A-d-r-i** [comment]
Thanks for your reply!

`The more common tag for this feature is highway=emergency_access_point - used 37k times`
Yes absolutely that's what I saw!  But is "highway = emergency_access_point" the most suitable?

You are absolutely right on the second point, I had not thought about the general use of the card.  I just imagined the usefulness of displaying this in an osmand-type cartography while hiking.

**Comment by DavidinHebron** [comment]
I hope this is the correct Forum to push this forward. If not, please point me in the right direction.

The tag: emergency_access_point is not visible on OSM Standard layer even at the highest zoom level - 19. 
I am trying to get it allowed (white-listed?) for the following reason.

I am in the USA. These markers are positioned at points on trails and paths so that Fire and Rescue can know for sure where someone is located. The use on OSM Standard layer is to plot the emergency_access_point on the map so that the ‘hiker’ (or someone in their party) can make their way to it to coordinate rescue. The Standard layer will show in any mobile browser without having to have previously downloaded tiles or apps, and is always up to date. The combination of the 'Show My Location' feature of the browser with visibility of the emergency_access_point provides a powerful aid for search and rescue.

In this use they are in sparsely used areas so there is not much map clutter - mostly just the trail and water bodies, but the intent was that they would only be visible at higher (19) zoom levels. I understand the issue with clutter but they only would be used in rural areas with a low density of geo-data.  OSM gatekeepers seem to be happy to display ‘waste baskets’ [amenity:waste_basket] in crowded city center displays [OpenStreetMap](https://www.openstreetmap.org/#map=18/48.1342991/11.5606542) at high zoom.
I suggest that the emergency_access_point has greater utility than a waste_basket.  The marker wouldn't clutter up the general-purpose map because it would only show up when zoomed in for hiking.

Thanks!

**Comment by imagico** [comment]
As @jeisenbe mentions - the problem is currently that there is apparently no agreement among mappers what the right tagging is, there are two competing tags.  The mapper community needs to sort this out before we can think about rendering here.

Beyond that here some numbers:

tag | world wide | Europe | Germany
----|----|----|----
`highway=emergency_access_point` |44k|42k|37k
`emergency=access_point` |11k|9.7k|5.4k

As you can see the distribution of use is very uneven world wide.  That does not rule out support in OSM-Carto but makes it somewhat low priority - this kind of feature seems to be only found map-worthy (or only exists) in small parts of the world, in contrast to waste baskets, which exist and are mapped world wide.

**Comment by DavidinHebron** [comment]
Thanks - I now see that this is a two stage issue, with getting consensus on which tag as the first step. 
It may be that they really are two similar but distinct things, where _highway=emergency_access_point_ is intended for roadside use say for identifying a breakdown location, and where emergency=access_point is intended for other situations. But I take your point.

These access points may only be used in a limited number of countries - Germany, Australia, Czech Republic and the USA, but these represent substantial populations and geographic areas.

Can you suggest the best Forum for discussing this issue with the mapping community to try and get agreement?
